addons/mrp_account/tests/test_valuation_layers.py

```diff
@@ -388,6 +389,45 @@ def test_validate_draft_kit(self):
         self.assertEqual(self.component.qty_available, 1)
         self.assertEqual(self.component.value_svl, 1424)
 
+    def test_average_cost_unbuild_with_byproducts(self):
+        """ Ensures that an unbuild for a manufacturing order using avg cost products won't
+            copy the value of the main product for every byproduct line, regardless of their real value
```

Comment:
Just to be under 100 characters everywhere :stuck_out_tongue: 
```suggestion
        """ Ensures that an unbuild for a manufacturing order using avg cost products won't copy
        the value of the main product for every byproduct line, regardless of their real value.
```